### ISSUE TITLE
TimeZoneConverter package installation failed on Xamarin.Forms PCL Project

### ISSUE THREAD

**Issue by ashwinmca2009** [mj1856/TimeZoneConverter#1]
TimeZoneConverter package installation failed in Xamarin forms PCL project. 

Please find the configuration details:
OS Version : Windows 10 
Visual studio 2015 with .Net framework 4.5

Error Description
```
Install failed. Rolling back...
Package 'TimeZoneConverter.1.0.0' does not exist in project 'eFACiLiTYMobile'
Install-Package : Could not install package 'TimeZoneConverter 1.0.0'. You are trying to install this package into a project that 
targets '.NETPortable,Version=v4.5,Profile=Profile78', but the package does not contain any assembly references or content files 
that are compatible with that framework. For more information, contact the package author.
At line:1 char:1
+ Install-Package TimeZoneConverter
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
```

**Comment by mj1856** [comment]
This should be resolved with release `1.1.0`.  Please let me know how it works for you.  Thanks.

**Comment by mj1856** [comment]
Actually, there's one additional thing.  Even though I fixed the nuget package, it is still targeting .NET Standard 1.1, because it needs the gzip support in `System.Compression` for the resource files.

You're targeting `Profile78` (from your error message), which is .Net Standard 1.0 - so this library won't work there.  You need to use a higher level profile, at minimum `Profile111`.  You can change this directly in your .csproj file.

See also:

https://docs.microsoft.com/en-us/dotnet/articles/standard/library#pcl-compatibility

http://danrigby.com/2014/04/16/xamarin-pcl-profile-notes/
